components/src/molecules/radio-wrapper/RadioWrapper.stories.js

```diff
@@ -0,0 +1,59 @@
+import { storiesOf } from '@storybook/vue'
+import { select } from '@storybook/addon-knobs'
+
+import ARadioWrapper from './RadioWrapper.vue'
+import ARadio from './../../atoms/radio/Radio.vue'
+
+const info = `
+  ---
+  Check **Knobs** tab to edit component properties dynamically. Below list of available BEM modifiers.
+  - \`.a-radio-wrapper--legend-hidden\` - Selector for hidding legend
+  ---
+`
+
+storiesOf('Molecules/Radio Wrapper', module)
+  .addParameters({ info })
+  .add('Default', () => ({
+    components: { ARadio, ARadioWrapper },
+    data () {
+      return {
+        selected: ''
+      }
+    },
+    props: {
+      classKnobs: {
+        default: select('BEM Modifier', [null, 'a-radio-wrapper--legend-hidden'])
+      }
+    },
+    template: `
+      <a-radio-wrapper
+        legend-text="Radio legend"
+        :class="classKnobs"
+      >
+        <a-radio
+          id="id-1"
+          value="option_one"
+          name="radio"
+          v-model="selected"
+        >
+          Option one
+        </a-radio>
+        <a-radio
+          id="id-2"
+          value="option_two"
+          name="radio"
+          v-model="selected"
+        >
+          Option two
+        </a-radio>
+        <a-radio
+          id="id-3"
+          value="option_three"
+          name="radio"
+          v-model="selected"
```

Comment:
I saw in vuetify and element support for `v-model` in radio-wrapper. In group of radio they specify one `v-model` in wrapper instead in each radio component https://codepen.io/pen/?&editable=true. Do you think we can do it too?

Reply:
I will create an issue for that